.github/workflows/macos.yml

```diff
@@ -93,7 +93,31 @@ jobs:
         with:
            submodules: recursive
 
-      - name: install boost
+      - name: install dependencies
+        run: |
+          brew install boost-build boost boost-python3 python@3.9
+
+      - name: build and install python module
+        run: |
+          cd bindings/python
+          /usr/local/opt/python@3.9/bin/python3.9 setup.py build_ext -j3 --libtorrent-link=static install
+
+      - name: run python tests
+        run: |
+          cd bindings/python
+          /usr/local/opt/python@3.9/bin/python3.9 test.py
+
+   python_b2:
```

Comment:
it looks like you preserved the existing test here. would you mind re-ordering these jobs to make the diff smaller, and reflect the fact that the existing test is unchanged?